src/sections/pools/modals/AddLiquidity/AddLiquidity.tsx

```diff
@@ -268,8 +268,18 @@ export const AddLiquidity = ({ pool, isOpen, onClose, onSuccess }: Props) => {
                     {t("liquidity.add.modal.warning")}
                   </Text>
 
-                  {isWithinLimit.data === false && <AddLiquidityLimitWarning />}
-
+                  {limits?.cap === false && (
```

Comment:
tady by se imo mel zobrazit jen jeden z techto erroru, v zadani se pise "Display whichever of them is lower"

Reply:
ano, ale funce ktera vraci cap limits je boolean hodnota, coz tam nejde zjistit lower hodnotu. Takze tam bude bud true nebo false